src/core/config.c

```diff
@@ -91,16 +91,20 @@ static char *get_config_path(void) {
 	char *config_home_fallback = calloc(size_fallback, sizeof(char));
 	snprintf(config_home_fallback, size_fallback, "%s/.config", home);
 
-	const char *prefix[3];
-	prefix[0] = getenv("XDG_CONFIG_HOME");
-	prefix[1] = config_home_fallback;
-	prefix[2] = SYSCONFDIR "/xdg";
+	const char *config_home = getenv("XDG_CONFIG_HOME");
+	if (config_home == NULL) {
```

Comment:
Nit: config_home could be empty (== ""?), if I read correctly.
>  $XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Reply:
Good catch! Fixed.